package.json

```diff
@@ -36,12 +36,12 @@
     "eslint-plugin-jsx-a11y": "6.2.3",
     "eslint-plugin-react": "7.14.2",
     "eslint-plugin-react-hooks": "1.6.1",
+    "fibers": "^4.0.2",
```

Comment:
Do we use fibers?

Reply:
[The documentation of `sass-loader` mentions `fibers`](https://github.com/webpack-contrib/sass-loader#implementation) and I had assumed that it speeds up compilation. But looking at it again I realised that I have no clue what they mean with “(a)synchronous compilation” and timing compilation with and without `fibers` I saw no difference, so I removed it again in b22a677c84e8.